### PR TITLE
Add v2.5.0 to release notes to CHANGELOG.md

### PR DESCRIPTION
**Pull request by yeazelm**
**Description of changes:**
Adds the release to changelog

**Testing done:**
NA

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by larvacea — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+* Build open source NVIDIA kernel modules ([#118])
+* Update third party packages ([#129], [#143])
+* Split ECS and VPC CNI plugins from ecs-agent ([#85])
+* Add helper functions for ipcidr ([#116])
+* Add pciutils to release ([#142])
+
+## Orchestrator Changes
+
+### Kubernetes
+* Use kubelet-device-plugin API) ([#132])
```
Comment (arnaldo2792) [inline CHANGELOG.md]:
```suggestion
* Use kubelet-device-plugins API) ([#132])
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
+# v2.5.0 (2024-09-11)
+
+## Build Changes
+* Update Tough (#[136])
```
Comment (jpculp) [inline CHANGELOG.md]:
```suggestion
* Update tough ([#136])
```

**Review by jpculp — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,31 @@
...
+* Update tough (#[136])
+* Update bottlerocket-sdk to v0.45.0 ([#131])
+
+## OS Changes
+* Build open source NVIDIA kernel modules ([#118])
+* Update third party packages ([#129], [#143])
+* Split ECS and VPC CNI plugins from ecs-agent ([#85])
+* Add helper functions for ipcidr ([#116])
+* Add pciutils package ([#142])
+
```
Comment (koooosh) [inline CHANGELOG.md:14]: We need to add change #50 : `* Add aws-otel-collector package ([#50])`

**Review by koooosh — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,33 @@
+# v2.5.0 (2024-09-11)
+
+## Build Changes
+* Update tough (#[136])
```
Comment (koooosh) [inline CHANGELOG.md]: Link is broken, should be: `([#136])`

**Review by koooosh — APPROVED** [review]

**Review by ginglis13 — APPROVED** [review]

**Review by jpculp — APPROVED** [review]

**Pull request merged**